README.md

```diff
@@ -10,8 +10,8 @@ ECMAScript Stage-0 Proposal. J. S. Choi, 2021.
 (A [formal specification][] is available.)
 
 **Method binding** `->` is a **left-associative infix operator**.
-Its right-hand side is an **identifier**
-or an **expression** in `(` `)`,
+Its right-hand side is an **identifier** (like `f`)
+or a parenthesized **expression** (like `hof()`),
```

Comment:
this is a function invocation, not a parenthesized expression (that would be an expression wrapped in parens)

Reply:
Ah, of course. I’ll wrap `hof()` in parentheses `(hof())`.

Reply:
Fixed in e8537951f21a8f2b3575c999003f3568018760d7.